gemfire/spring-cloud-starter-stream-source-gemfire/src/main/resources/gemfire-source.yml

```diff
@@ -1,4 +1,6 @@
 #
 # Gemfire client pool must be configured with subscriptions enabled
 #
-subscriptionEnabled:true
+gemfire:
+    pool:
+        subscriptionEnabled:true
```

Comment:
How about this concern?
It is really bug. I won't pass PR without the fix! :smile: